sigstore/sign.py

```diff
@@ -238,6 +239,12 @@ def _to_bundle(self) -> Bundle:
                 hashes=[
                     bytes.fromhex(h) for h in self.log_entry.inclusion_proof.hashes
                 ],
+                checkpoint=Checkpoint(
+                    envelope=self.log_entry.inclusion_proof.checkpoint or ""
+                )
```

Comment:
This fallback shouldn't be necessary anymore, right?

```suggestion
                checkpoint=Checkpoint(
                    envelope=self.log_entry.inclusion_proof.checkpoint
                )
```